.github/workflows/studio-release-tests.yml

```diff
@@ -19,7 +17,7 @@ jobs:
           pip install -r requirements.txt
           pip install -r requirements/requirements.test.txt
           pip install -e .
-          pip install coverage coveralls
+          pip install coverage
```

Comment:
pretty sure we don't use coveralls

Reply:
I don't get this one, though: if we don't install coveralls, suddenly `--omit g2p/tests/*` stops working and we get that -43% drop in coverage you saw.

So we don't use coveralls, but we need its side effect. I've done additional tests and I did not come to understand why, but I positively confirmed just installing coveralls or not is what determines if tests are ommited from coverage or not.